content/cloud/variables.md

```diff
@@ -0,0 +1,191 @@
+title = "Configuring Spin Application Variables and Secrets"
+date = "2023-06-05T16:40:01.672734Z"
+template = "cloud_main"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main//content/cloud/variables.md"
+
+---
+
+- [Prerequisites](#prerequisites)
+- [Creating Our Spin Application](#creating-our-spin-application)
+- [Adding Variables to an Application Component](#adding-variables-to-an-application-component)
+- [Using Variables in a Spin Application](#using-variables-in-a-spin-application)
+- [Deploying the Application to Fermyon Cloud](#deploying-the-application-to-fermyon-cloud)
+
+Spin supports dynamic application variables. Instead of being static, their values can be updated without modifying the application, creating a simpler experience for rotating secrets, updating API endpoints, and more. 
+
+These variables are defined in a Spin application manifest (in the `[variables]` section) and are provided by a configuration provider. When using Spin locally, the configuration provider can be Vault for secrets or host environment variables. You can refer to the [dynamic configuration documentation](../spin/dynamic-configuration.md) to learn how to configure variables locally. In Fermyon Cloud, you can set and update variables for Spin applications using the spin cloud variables CLI.
```

Comment:
"spin cloud variables CLI"

That sounds like a particular CLI I need. I would recommend rewording to "the `spin cloud variables` command, from the Fermyon Cloud Plug-in. And maybe make a reference to a doc about the Fermyon Cloud Plug-in.

Reply:
Good idea on specifying its a subcommand. We dont have or plan to have a doc on the plugin. I am not sure how much it is necessary that users know about it since their experience is the same as before

Reply:
now that we are working on adding a page for the cloud command reference, i added a link that will work once it is merged.